Include/internal/pycore_pystate.h

```diff
@@ -132,6 +132,15 @@ struct _is {
     struct _warnings_runtime_state warnings;
 
     PyObject *audit_hooks;
+/*
+ * See bpo-36876: miscellaneous ad hoc statics have been moved here.
+ */
+    struct {
+        struct {
+            int level;
+            int atbol;
+        } listnode_data;
+    } parser_data;
```

Comment:
Likewise with "_data".

```suggestion
    } parser;
```

Reply:
Done.